### ISSUE TITLE
Nextcloud-Agent crashed during download

### ISSUE THREAD

**Issue by strangmann** [nextcloud/android#13171]
### Cause of error
```java
Exception in thread "WM.task-2" kotlin.UninitializedPropertyAccessException: lateinit property notificationManager has not been initialized
    at com.nextcloud.client.jobs.download.FileDownloadWorker.onStopped(FileDownloadWorker.kt:154)
    at androidx.work.ListenableWorker.stop(ListenableWorker.java:294)
    at androidx.work.impl.WorkerWrapper.interrupt(WorkerWrapper.java:386)
    at androidx.work.impl.Processor.interrupt(Processor.java:440)
    at androidx.work.impl.Processor.stopWork(Processor.java:260)
    at androidx.work.impl.utils.StopWorkRunnable.run(StopWorkRunnable.kt:51)
    at androidx.work.impl.utils.SerialExecutorImpl$Task.run(SerialExecutorImpl.java:96)
    at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
    at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:644)
    at java.lang.Thread.run(Thread.java:1012)

```

### App information
* ID: `com.nextcloud.client`
* Version: `30290090`
* Build flavor: `gplay`

### Device information
* Brand: `samsung`
* Device: `gts8pwifi`
* Model: `SM-X800`
* Id: `UP1A.231005.007`
* Product: `gts8pwifieea`

### Firmware
* SDK: `34`
* Release: `14`
* Incremental: `X800XXS7CXE9`

**Comment by strangmann** [comment]
Missing Information from Issue-Template:
- Actual behaviour: I was synchronising a directory of MP4 files and while doing so, I watched one of them
- Nextcloud-Server Version: 28.0.6
- Not using a reverse-proxy

**Comment by joshtrichards** [comment]
Fixed in #13073
